splunk-forwarder/README.md

```diff
@@ -0,0 +1,17 @@
+# Splunk forwarder for Cloud Watch Log group subscriptions
+
+This module lets you forward logs from Cloud Watch Logs in your account to Splunk. The module creates Cloud Watch subscriptions Log groups, and targets the subscription to a Lambda function.
+
+[The Lambda artifact can be found here](https://github.com/TeliaSoneraNorge/cloudwatch-subscriptions-s3-writer)
+
+## Architectural overview 
+
+![alt text][diagram]
```

Comment:
FYI; This seems borked now

Reply:
Fixed

Reply:
Hm, does not render on my end: `![alt text][diagram] [diagram]: img/architecture.png "Architecture overview"` <- seems wrong?